src/pages/en/guides/deploy/github.md

```diff
@@ -70,12 +70,15 @@ You can deploy an Astro site to GitHub Pages by using [GitHub Actions](https://g
     This workflow uses the `npm ci` command by default. You must include a `package-lock.json` file in your repository for this to work. To generate one, run `npm i` in your terminal and commit the resulting lock file.
     :::
 
-    :::tip
-    See [the GitHub Pages Action documentation](https://github.com/marketplace/actions/github-pages-action) for different ways you can configure the final “Deploy to GitHub Pages” step.
-    :::
 
-3. Commit the new workflow file and push it to GitHub.
-4. On GitHub, go to your repository’s **Settings** tab and find the **Pages** section of the settings.
-5. Choose the `gh-pages` branch and the `"/" (root)` folder as the **Source** of your site and press **Save**.
+3. Commit the new workflow file and push it to GitHub.  
+
+4. On GitHub, go to your repository’s **Settings** tab and find the **Pages** section of the settings.  
+
+5. Choose the `gh-pages` branch and the `"/" (root)` folder as the **Source** of your site and press **Save**.  
+  
 
+:::tip
+If you want a custom domain rather than `user.github.io`, follow [the official Github Guide for configuring domains](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-a-subdomain) and make sure to place the `CNAME` file into your `public` folder.
+:::
```

Comment:
```suggestion
:::tip
If you want a custom domain rather than `user.github.io`, create a file named `CNAME` (no extension) in your `public` folder. The content of the file should be just your Domain like `example.com` or `subdomain.example.com`
:::
```

Reply:
I will point out that you made this change in the first place because you thought we didn't need instructions here since there was a link to GH's instructions! 

Even if GH's instructions are confusing, having a link to them does provide help to the reader, especially being the official documentation for what they are trying to achieve. I think this takes too much away from the reader.

Reply:
I think we might be diving a little off the purpose of the deploy guide. My understanding was that this guide was to help the user publish their project online. 

I like the idea of having an aside or tip to point out further configuration to the respective documentation guides.

Reply:
The problem with the GH Docs are that they just don't make it clear that much because they use a different approach then we have here. The problem originally was more that it was a lot to read before going on with the main steps. That's why I trimmed it down.